### ISSUE TITLE
Support Typescript

### ISSUE THREAD

**Issue by dorozco-dev** [Akifcan/googlemaps-vue-3#5]
I am doing a development in vue 3 with typescript but it is not supported

**Comment by Khel** [comment]
@dorozco-dev Im also using this with typescript. I have created a googlemaps.d.ts file with the statement declare module "googlemaps-vue3"; That, at least gets rid of the missing types error.
Im currently trying to insert it in a view but it says ./node_modules/googlemaps-vue3/GoogleMap.js
Module not found: Error: Can't resolve '@/components/GoogleMap.vue' in 'C:\wamp64\www\Collaborativeperks\OnlyPerks\Frontend\Client\onlyperks-frontend-base-aux\node_modules\googlemaps-vue3'

Still on it :)

**Comment by curlylazy** [comment]
yes me too, i'm facing problem with " Can't resolve '@/components/GoogleMap.vue' in"..
